src/components/GoogleAddressLookup/component.js

```diff
@@ -237,6 +239,7 @@ PlacesLookupComponent.propTypes = {
     onClick: PropTypes.func,
     // onFocus: PropTypes.func,
     onBlur: PropTypes.func,
+    variant: PropTypes.oneOf(['default', 'shaded', 'bare']),
```

Comment:
this should be added to the index.d.ts too, in order to support it in typescript